src/components/Modals/CreateSingleItemModal/ImportStep/ImportStep.tsx

```diff
@@ -100,7 +108,21 @@ export default class ImportStep extends React.PureComponent<Props, State> {
       [modelPath]: file
     }
 
-    return this.processModel(modelPath, contents)
+    const { model, contents: proccessedContent, type } = await this.processModel(modelPath, contents)
+
+    if (type === ItemType.EMOTE) {
+      const data = await getItemData({
+        wearablePreviewController: WearablePreview.createController('wearable-preview'),
```

Comment:
is there a way we can re-use the controller that we're receiving by parameter? It's wrapped in a React component but we could change how it's being passed down to be just the controller and wrap it here. WDYT?

Reply:
sounds good! I'll refactor the wrapped component to use here :D